content/en/docs/instrumentation/net/automatic/nuget-packages.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Using the OpenTelemetry.AutoInstrumentation NuGet packages
+linkTitle: NuGet Packages
+weight: 40
+cSpell:ignore: autoinstrumentation buildtasks
+---
+
+Use the NuGet packages in the following scenarios:
+
+1. Simplify deployment. For example, a container running a single application.
+1. Support instrumentation of
+   [`self-contained`](https://learn.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained)
+   applications.
+1. Facilitate developer experimentation with automatic instrumentation through
+   NuGet packages.
+1. Solve version conflicts between the dependencies used by the application and
+   the automatic instrumentation.
+
+## Limitations
+
+While NuGet packages are a convenient way to deploy automatic instrumentation,
+they can't be used in all cases. The most common reasons for not using NuGet
+packages include the following:
+
+1. You can't add the package to the application project. For example, the
+   application is from a third party that can't add the package.
+1. Reduce disk usage, or the size of a virtual machine, when multiple
+   applications to be instrumented are installed in a single machine. In this
+   case you can use a single deployment for all .NET applications running on the
+   machine.
+1. A legacy application that can't be migrated to the
```

Comment:
Erhm, is this actually correct though? So long as you're targeting a .NET Standard-compatible framework version (4.7.2 or higher), you can use nuget packages.

I think the real difference is when a package is targeting netcoreapp or net5+.

Reply:
Unfortunately, the statement is correct. The issue is that there is a native component and SDK style gets the correct one from the package without any "tricks". To get that to work with old style projects we will need to add the native dll as content instead of the runtime. It is doable with some clever build targets embedded into the NuGet. If there is demand for that we can look into it, however, it doesn't work right now.

Reply:
Ahhh, I see. Whew, I had forgotten about how hellish packaging is for non-SDK style projects 🙃 

In that case, we should clarify the issue related to including native components, as that's the current reason.

Reply:
On it